### PR TITLE
light: fix various linter errors

### PR DESCRIPTION
**Pull request by mitzkia**
There were so many linter errors, most of them I have fixed for now.

Later we can decide to add constant linter checkers for light code also.

This PR depends on my previous: #2587

### CODE REVIEW

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by kira-syslogng** [comment]
Build FAILURE

**Comment by Kokan** [comment]
@kira-syslogng retest this please

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by mitzkia** [comment]
I have rebased this PR to master, so #2587 is no more dependency here.

**Comment by kira-syslogng** [comment]
Build SUCCESS

File: tests/python_functional/src/message_reader/message_reader.py [hunk 1]
```diff
@@ -20,9 +20,8 @@
 # COPYING for details.
 #
 #############################################################################
-
-from src.common.blocking import wait_until_true_custom
 from src.common.blocking import DEFAULT_TIMEOUT
+from src.common.blocking import wait_until_true_custom
```
Comment (furiel) [inline tests/python_functional/src/message_reader/message_reader.py]: What is the idea here? In `test_message_reader.py` the order is the opposite (`READ_ALL_MESSAGES` vs `MessageReader`)

Comment (mitzkia) [reply tests/python_functional/src/message_reader/message_reader.py]: The rule is: they are in alphabetical order

**Pull request merged**